static/app/views/performance/newTraceDetails/traceTree.tsx

```diff
@@ -152,7 +152,8 @@ function maybeInsertMissingInstrumentationSpan(
     return;
   }
 
-  if (node.value.start_timestamp - lastInsertedSpan.value.timestamp < 100) {
+  const gapInMs = (node.value.start_timestamp - lastInsertedSpan.value.timestamp) * 1000;
```

Comment:
There is no need to compute this. If the value is in seconds, just check that difference is < 0.1